cache/redis.go

```diff
@@ -85,8 +104,25 @@ func (r *redisClusterClient) Set(ctx context.Context, key string, val string, tt
 	return r.client.Set(ctx, key, val, ttl).Err()
 }
 
-func (r *redisClusterClient) Get(ctx context.Context, key string) (string, error) {
-	val, err := r.client.Get(ctx, key).Result()
+func (r *redisClusterClient) Incr(ctx context.Context, key string, subtract bool) (int, error) {
```

Comment:
I'm not familiar enough with Go to know about this - but logic is the same as above in redisSimpleClient. Is there a polymorphic way to use the same code? (ex: can redisClusterClient inherit and not override Incr/Get)

Reply:
perhaps there is a way, but might be best to follow up once things are stable